### PR TITLE
Webextension stack tests

### PR DESCRIPTION
**Pull request by chartjes**
These tests are shell scripts that talk to the Kinto end points that webextensions is using. It verifies that data can be read and written.

r? @rpappalax

### CODE REVIEW

**Comment by rpappalax** [comment]
@chartjes can you clarify how these will be employed?  If you're planning to use them mostly manually for now, then I'd recommend putting them in a subdir called "scripts."  That way they won't be confused as being used for CI.  
If you're planning on adding these to the CI pipeline (recommended for use with deploys), you'd need to commit a change to the run file to make sure they get invoked.  
Also, is there a reason why you wrote those tests in bash vs. python?  In general, my impression is that bash is better used for boot-strapping but, in this case, these are your actual tests.

File: config-test/webextensions-check-prod.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -x
+
+export FXA_USER_EMAIL="chartjes@mozilla.com"
+
+export SERVER_URL="https://webextensions.settings.services.mozilla.com/v1"
```
Comment (rpappalax) [inline config-test/webextensions-check-prod.sh]: where possible, i'd still recommend moving your constants off to a manifest.ini file. things like your user email or server DNS we're putting into manifest files with clearly marked sections for [dev]  [stage] [prod], that way the tests can easily be invoked with different test envs and parameters.

Comment (chartjes) [inline config-test/webextensions-check-prod.sh]: I do not know how to read in a manifest.ini file from a shell script.

**Comment by chartjes** [comment]
@rpappalax These scripts are meant to be run after the service has been deployed to stage or production. I figured they needed to be run manually because it requires a password for a Firefox account to be entered so we get a bearer token.

I spent an entire day trying to duplicate what the shell script does as a Python script and could not duplicate the results using existing libraries. I can give it another pass but we need something *right now* to test things post-deployment.

I can move them to a different directory for the time being.

**Pull request merged**